packages/server/express/src/server.ts

```diff
@@ -60,8 +60,9 @@ const defineAPI = async function (config: IConfig, storage: Storage): Promise<Ex
   );
 
   // Hook for tests only
-  if (config._debug) {
-    hookDebug(app, config.configPath);
+  if (config._debug && app.get('env') !== 'production') {
```

Comment:
`!== 'production'` 🤔 not sure if this might create issues in our E2E for docker I'm considering to build, could we exclude it for now?